dev/gpctl/cmd/public-api.go

```diff
@@ -51,7 +51,7 @@ func newPublicAPIConn() (*grpc.ClientConn, error) {
 	opts := []grpc.DialOption{
 		// attach token to requests to auth
 		grpc.WithUnaryInterceptor(func(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
-			withAuth := metadata.AppendToOutgoingContext(ctx, "authorization", publicApiCmdOpts.token)
+			withAuth := metadata.AppendToOutgoingContext(ctx, "authorization", "Bearer "+publicApiCmdOpts.token)
```

Comment:
Was not correctly adding the Bearer prefix, and we weren't also correctly checking it for it server side.